docs/token-contracts/fa2/2-fa2-nft-tutorial.md

```diff
@@ -0,0 +1,533 @@
+
+The commands will be explained in more detail below. You can always run
+
+```sh
+$ tznft --help
+```
+
+to list all available commands.
+
+### Initial Setup
+
+1. Create a new local directory to keep your tutorial configuration:
+
+   ```sh
+   $ mkdir nft-tutorial
+   $ cd nft-tutorial
+   ```
+
+2. Install `@tqtezos/nft-tutorial` npm package:
+
+   ```sh
+   $ npm install -g https://github.com/tqtezos/nft-tutorial.git
+
+   /usr/local/bin/tznft -> /usr/local/lib/node_modules/@tqtezos/nft-tutorial/lib/tznft.js
+
+   + @tqtezos/nft-tutorial@1.0.0
+   added 3 packages from 1 contributor and updated 145 packages in 11.538s
+   ```
+
+   The command installs `tznft` CLI tool.
+
+3. Initialize tutorial config:
+
+   ```sh
+   $ tznft init-config
+
+   tznft.json config file created
+   ```
+
+4. Check that the default active network is `sandbox`:
+
+   ```sh
+   $ tznft show-network
+
+   active network: sandbox
+   ```
+
+5. Bootstrap Tezos network:
+
+   ```sh
+   $ tznft bootstrap
+
+   ebb03733415c6a8f6813a7b67905a448556e290335c5824ca567badc32757cf4
+
+   starting sandbox...
+   sandbox started
+   originating balance inspector contract...
+   originated balance inspector KT1Pezr7JjgmrPcPhpkbkH1ytG7saMZ34sfd
+   ```
+
+   If you are bootstrapping a `sandbox` network for the first time, Docker will download
+   the Flextesa docker-image as well.
+
+   The default configuration comes with two account aliases `bob` and `alice`
+   that can be used for token minting and transferring.
+
+### Mint NFT Token(s)
+
+This tutorial uses an NFT collection contract. Each time the user mints a new set
+(collection) of tokens, a new NFT contract is created. The user cannot add more
+tokens or remove (burn) existing tokens within the contract. However tokens can
+be transferred to other owners.
+
+`mint` command requires the following parameters:
+
+- <owner> alias or address of the new tokens owner
+- `--tokens` new tokens metadata. Each token metadata is represented as comma
+  delimited string: `'<token_id>, <token_symbol>, <token_name>'`:
+
+```sh
+$ tznft mint <owner_alias> --tokens <token_meta_list>`
+```
+
+Example:
+
+```sh
+$ tznft mint bob --tokens '0, T1, My Token One' '1, T2, My Token Two'
+
+originating new NFT contract...
+originated NFT collection KT1XP3RE6S9t44fKR9Uo5rAfqHvHXu9Cy7fh
+```
+
+### Inspecting The NFT Contract
+
+Using `KT1..` address of the NFT contract created by the `mint` command, we can
+inspect token metadata and balances (i. e. which addresses own the tokens).
+
+#### Inspect Token Metadata
+
+`show-meta` command requires the following parameters:
+
+- `--nft` address of the FA2 NFT contract to inspect
+- `--signer` alias on behalf of which contract is inspected
+- `--tokens` a list of token IDs to inspect
+
+```sh
+$ tznft show-meta --nft <nft_address> --signer <alias> --tokens <token_id_list>
+```
+
+Example:
+
+```sh
+$ tznft show-meta --nft KT1XP3RE6S9t44fKR9Uo5rAfqHvHXu9Cy7fh --signer bob --tokens 0 1
```

Comment:
Could you replace commands involving addresses, e.g. `KT1XP3RE6S9t44fKR9Uo5rAfqHvHXu9Cy7fh` with bash variables? This makes it much easier to copy/paste commands without editing each one.